test/utils/image/manifest.go

```diff
@@ -114,6 +114,8 @@ const (
 	CudaVectorAdd
 	// CudaVectorAdd2 image
 	CudaVectorAdd2
+	// CudaVectorAdd3 image
+	CudaVectorAdd3
```

Comment:
instead of adding a new variable for every version, can't we just bump the version for `CudaVectorAdd` itself to point to the latest version? do we have any scenario to test all the supported versions? if yes - then that should go to the test metrics for cuda itself.

Reply:
@mkumatag For GPU, it is important to make sure we support backward compatibility, i.e., containers built with old Cuda version can continue to run successfully. Generally, new cuda versions are not released frequently, but I understand your concern on keeping adding new variables for every version. Do you have any specific suggestion on how we may avoid this from cuda side?

Reply:
I think we need to understand the scope of the testing here! If it is about testing some GPU device plugin and sample workload is fine. But it is about testing cuda library with multiple versions will become more of a functionality test for Cuda and device plugin(GoogleCloudPlatform/container-engine-accelerators) driver for some extent.

And even if it is faulty or regression issues I feel the change will end up going into either cuda library or `GoogleCloudPlatform/container-engine-accelerators`, so IMO we need to add more tests under `GoogleCloudPlatform/container-engine-accelerators` bucket.

Reply:
Thanks for bringing this concern up @mkumatag.

What do you think if we keep the current CudaVectorAdd and CudaVectorAdd2 and instead of adding additional images to test for new CUDA version, we rotate the images so when we update for new CUDA versions, the tagged images will rotate out. 
cuda-vector-add:2.0 <- updated image with new CUDA version
cuda-vector-add:1.0 <- previous cuda-vector-add:2.0

This way we do not need to add increasing number of images, and can also ensure the latest CUDA version is supported and some degree of backward compatibility.

Reply:
> This way we do not need to add increasing number of images, and can also ensure the latest CUDA version is supported and some degree of backward compatibility.

I'm good with this approach. 👍

Reply:
Thanks, I'll go ahead and make that change.

Reply:
Updated so that on push, there is a check for PREVIOUS_VERSION and if exists, it will rename the existing latest version image.